### PR TITLE
{2023.06} EasyBuild v4.8.1 + re-install Java/11 with it

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
bot: build repo:eessi-2023.06-software arch:x86_64/generic
bot: build repo:eessi-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi-2023.06-software arch:aarch64/generic
bot: build repo:eessi-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi-2023.06-software arch:aarch64/neoverse_v1

**Comment by bedroge** [comment]
In EB 4.8.1 `Java/11` was bumped to `Java-11.0.20.eb`, and that's now missing:
```
1 out of 2 required modules missing:
* Java/11.0.20 (Java-11.0.20.eb)
```
So we need to (re)install Java.

File: eessi-2023.06-eb-4.8.1-system.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+easyconfigs:
+  - Java-11.eb:
+      options:
+        rebuild: True
```
Comment (boegel) [inline eessi-2023.06-eb-4.8.1-system.yml:4]: Note: this should be removed before merging this PR!

**Comment by boegel** [comment]
bot: build repo:eessi-2023.06-software arch:x86_64/generic
bot: build repo:eessi-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi-2023.06-software arch:aarch64/generic
bot: build repo:eessi-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi-2023.06-software arch:aarch64/neoverse_v1

**Review by bedroge — APPROVED** [review]

**Pull request merged**